helm/charts/oathkeeper/values.yaml

```diff
@@ -111,3 +111,6 @@ tolerations: []
 
 # Configure node affinity
 affinity: {}
+
+# Annotations
+annotations: {}
```

Comment:
It should be clear that these are for the deployment, otherwise I would probably expect these annotations to be merged everywhere (e.g. ingress etc)

Reply:
moved all relevant options into a `deployment` block